mcprep-build.py

```diff
@@ -0,0 +1,101 @@
+import os
+import shutil
+import sys
+from pathlib import Path
+from typing import List
+
+# Config stuff
+BUILD_NAME: str = "MCprep_addon"
+
+# Internals
+BUILD_DIRECTORY: Path = Path("build")
+INTERMIDIATE_PATH: Path = BUILD_DIRECTORY / Path("inter")
+ADDON_DIRECTORY: Path = Path("MCprep_addon")
+BLENDER_INSTALLS: Path = Path("blender_installs.txt")
```

Comment:
May as well also define BLENDER_EXECS, to help reinforce the concept.

Reply:
I dunno, this is mostly a building script so I don't think it makes sense to add BLENDER_EXECS